src/core/main.c

```diff
@@ -801,6 +803,7 @@ static int parse_argv(int argc, char *argv[]) {
                 { "default-standard-output",  required_argument, NULL, ARG_DEFAULT_STD_OUTPUT,      },
                 { "default-standard-error",   required_argument, NULL, ARG_DEFAULT_STD_ERROR,       },
                 { "machine-id",               required_argument, NULL, ARG_MACHINE_ID               },
+                { "root-propagation-shared",  no_argument,       NULL, ARG_ROOT_PROPAGATION_SHARED  },
```

Comment:
We'll need to document this in https://github.com/systemd/systemd/blob/master/man/systemd.xml